config/config.go

```diff
@@ -517,6 +521,10 @@ func NewConfig() *Config {
 	// TLS version
 	flag.StringVar(&cfg.TLSMinVersion, "tls-min-version", defaultMinTLSVersion, "minimal TLS Version to be used in server, proxy and client connections")
 
+	// Cipher Suites
+	flag.BoolVar(&cfg.ExcludeInsecureCipherSuites, "exclude-insecure-cipher-suites", false, "excludes insecure cipher suites")
+	flag.StringVar(&cfg.AllowedCipherSuites, "allowed-cipher-suites", "", "comma separated list of exclusively allowed cipher suites")
```

Comment:
Please check listFlag and multiFlag types

Reply:
I think we should start with a single boolean flag to disable insecure suites and then add a list flag if we even find a need for it.